tests/unit/swagger/schemaDetails3.spec.ts

```diff
@@ -791,6 +791,12 @@ describe('Definition generation for OpenAPI 3.0.0', () => {
           });
         });
 
+        it('Should not generate models with hidden controller referenced.', () => {
+          const metadata = new MetadataGenerator('./fixtures/controllers/commonResponseHiddenModelController.ts').Generate();
```

Comment:
Should still be in the metadata, which at least implicated that the route and validation will exist.